src/go/k8s/pkg/resources/configmap.go

```diff
@@ -170,6 +171,14 @@ func (r *ConfigMapResource) createConfiguration() *config.Config {
 		},
 	}
 	cr.Directory = dataDirectory
+	if r.pandaCluster.Spec.Configuration.TLS.KafkaAPIEnabled {
+		cr.KafkaApiTLS = config.ServerTLS{
+			KeyFile:	fmt.Sprintf("%s/%s", tlsDir, corev1.TLSPrivateKeyKey),	// tls.key
+			CertFile:	fmt.Sprintf("%s/%s", tlsDir, corev1.TLSCertKey),	// tls.crt
+			TruststoreFile:	fmt.Sprintf("%s/ca.crt", tlsDir),
```

Comment:
```suggestion
			TruststoreFile:	fmt.Sprintf("%s/%s", tlsDir, CAKey),
```
In the `certificate.go` there is const defined

Reply:
thanks